CMakeLists.txt

```diff
@@ -7,5 +7,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 project(risc_v_simulator)
+find_package(GTest CONFIG REQUIRED)
```

Comment:
Change this line to `add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/third-party/googletest)` help me to build project

Reply:
Will be fixed in other PR